### PR TITLE
feat(select): add accessibility to select component

### PR DESCRIPTION
**Pull request by blackswanny**
add accessibility to select component

### CODE REVIEW

File: src/select/styled-components.js [hunk 1]
```diff
@@ -152,3 +152,7 @@ export const SelectSpinner = styled('div', () => {
     paddingLeft: '50%',
   };
 });
+
+export const SelectionList = styled('div', props => {
```
Comment (nadiia) [inline src/select/styled-components.js]: How about we call it `SelectionContainer`? Since it's not a list.

File: rfcs/select-component.md [hunk 2]
```diff
@@ -177,6 +178,7 @@ export default () => {
   * `SelectComponentIcon: ?React.ComponentType` component for all icons appearing in Select component. It's provided `$type: ICON.loop | ICON.clearTag | ICON.clearAll | ICON.selected` to setup corresponding icon of Select component
   * `Tag: ?React.ComponentType` component for selected options Tags shown in Input for multiple mode selection
   * `SelectSpinner: ?React.ComponentType` component for dropdown spinner if async options are loading
+  * `SelectionList: ?React.ComponentType` component for the wrapper around selected options\tags
```
Comment (nadiia) [inline rfcs/select-component.md]: Commented below regarding the name.

File: src/select/styled-components.js (hunk 1 above)
Comment (blackswanny) [reply src/select/styled-components.js]: ok

File: rfcs/select-component.md (hunk 2 above)
Comment (blackswanny) [reply rfcs/select-component.md]: ok

**Pull request merged**